platform/binance/model_test.go

```diff
@@ -0,0 +1,55 @@
+package binance
+
+import (
+	"reflect"
+	"testing"
+)
+
+var newOrderDataDst = Data{OrderData: OrderData{
+	Symbol: "AWC-986_BNB",
+	Base:   "AWC-986",
+	Quote:  "BNB",
+	Price:  0.00324939,
+}}
+
+var cancelOrderDataDst = Data{OrderData: OrderData{
+	Symbol: "GTO-908_BNB",
+	Base:   "GTO-908",
+	Quote:  "BNB",
+	Price:  0.00104716,
+}}
+
+func TestTx_getData(t *testing.T) {
+	tests := []struct {
+		name    string
+		Data    string
+		want    Data
+		wantErr bool
+	}{
+		{
+			"new order",
+			"{\"orderData\":{\"symbol\":\"AWC-986_BNB\",\"orderType\":\"limit\",\"side\":\"buy\",\"price\":0.00324939,\"quantity\":2.00000000,\"timeInForce\":\"GTE\",\"orderId\":\"D13BAF4BD6638FA3AAD6EBCA0E4BEEA73DF4D519-30\"}}",
```

Comment:
@Pantani should we suggest binance dex folks to have this as an object and not as a string?

Reply:
definitely yes 👍